group1/comm/tests.py

```diff
@@ -1,3 +1,34 @@
 from django.test import TestCase
+from comm.models import Message, Room
+from datetime import datetime
+from django.contrib.auth.models import User
 
 # Create your tests here.
+class RoomTestCase(TestCase):
+    def setUp(self):
+    	#Create room and user objects
+        Room.objects.create(name = "test1", description = "test1")
+        room1 = Room.objects.get(name = "test1")
+        Room.objects.create(name = "test2", description = "test2")
+        room2 = Room.objects.get(name = "test2")
```

Comment:
More of a question than feedback: why do we need two test rooms here? Can't we just do the assertions on one room? If we have a different test case in which we want to keep track of message counts from multiple rooms then it might be good to implement that in a different test function.